src/commands/rate.ts

```diff
@@ -0,0 +1,85 @@
+import { LanguageKeys } from '#lib/i18n/LanguageKeys';
+import { escapeMarkdown } from '@discordjs/builders';
+import { envParseArray } from '@skyra/env-utilities';
+import { Command, RegisterCommand } from '@skyra/http-framework';
+import { applyLocalizedBuilder, getT } from '@skyra/http-framework-i18n';
+
+const Root = LanguageKeys.Commands.Rate;
+
+@RegisterCommand((builder) =>
+	applyLocalizedBuilder(builder, Root.RootName, Root.RootDescription) //
+		.addStringOption((builder) => applyLocalizedBuilder(builder, Root.OptionsRateableTarget).setRequired(true))
+)
+export class UserCommand extends Command {
+	private readonly oneToTenMap = new Map<number, UtilOneToTenEntry>([
+		[0, { emoji: '😪', color: 0x5b1100 }],
+		[1, { emoji: '😪', color: 0x5b1100 }],
+		[2, { emoji: '😫', color: 0xab1100 }],
+		[3, { emoji: '😔', color: 0xff2b00 }],
+		[4, { emoji: '😒', color: 0xff6100 }],
+		[5, { emoji: '😌', color: 0xff9c00 }],
+		[6, { emoji: '😕', color: 0xb4bf00 }],
+		[7, { emoji: '😬', color: 0x84fc00 }],
+		[8, { emoji: '🙂', color: 0x5bf700 }],
+		[9, { emoji: '😃', color: 0x24f700 }],
+		[10, { emoji: '😍', color: 0x51d4ef }]
+	]);
+
+	private readonly devRegex = new RegExp(
+		`^(kyra|favna|${envParseArray('OWNER_IDS')
+			.map((owner) => `<@!?${owner}>`)
+			.join('|')})$`,
+		'i'
+	);
+
+	private readonly botRegex = new RegExp(`^(you|yourself|artiel|<@!${process.env.CLIENT_ID}>)$`, 'i');
```

Comment:
```suggestion
	private readonly botRegex = new RegExp(`^(you|yourself|artiel|<@!?${process.env.CLIENT_ID}>)$`, 'i');
```

Same as the regex above, the `!` in the user mention should be optional.